cdap-api/src/main/java/co/cask/cdap/api/app/AbstractApplication.java

```diff
@@ -170,4 +173,16 @@ protected void scheduleWorkflow(Schedule schedule, String workflowName) {
   protected void scheduleWorkflow(Schedule schedule, String workflowName, Map<String, String> properties) {
     configurer.addSchedule(schedule, SchedulableProgramType.WORKFLOW, workflowName, properties);
   }
+
+
+  /**
+   * Schedule the specified {@link Workflow}
```

Comment:
Do you have an example of what this will look like in application code? I feel that calling it addSchedule() would make sense... Perhaps we can have something like:
```
  addSchedule("mySchedule")
    .forWorkflow("wf1")
    .setDescription("my schedule")
    .addConstraint(...)
    .triggerOn(...);
```